From a6f767fa6e1b15bd75142c4cf93d51f21887d75b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Lureau?= <marcandre.lureau@gmail.com>
Date: Fri, 17 Feb 2012 17:29:25 +0100
Subject: [PATCH 1/3] Add controller message ENABLE_SMARTCARD

The sent message must be a ControllerValue whose value is either '0'
disabled, or '1' enabled.
---
 configure.ac            |    2 +-
 spice/controller_prot.h |    2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index e1c3861..9047742 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2,7 +2,7 @@ AC_PREREQ([2.57])
 
 m4_define([SPICE_MAJOR], 0)
 m4_define([SPICE_MINOR], 10)
-m4_define([SPICE_MICRO], 1)
+m4_define([SPICE_MICRO], 2)
 
 AC_INIT(spice-protocol, [SPICE_MAJOR.SPICE_MINOR.SPICE_MICRO], [], spice-protocol)
 
diff --git a/spice/controller_prot.h b/spice/controller_prot.h
index 44d78e1..a60b122 100644
--- a/spice/controller_prot.h
+++ b/spice/controller_prot.h
@@ -73,6 +73,8 @@ enum {
     CONTROLLER_SHOW,
     CONTROLLER_HIDE,
 
+    CONTROLLER_ENABLE_SMARTCARD,
+
     //spice client -> extrenal app
     CONTROLLER_MENU_ITEM_CLICK = 1001,
 };
-- 
1.7.9.3

